admin_manual/ai/app_translate.rst

```diff
@@ -0,0 +1,93 @@
+==========================================
+App: Local Machine translation (translate)
+==========================================
+
+.. _ai-app-translate:
+
+The *translate* app is one of the apps that provide machine translation functionality in Nextcloud and act as a translation backend for the :ref:`Nextcloud Assistant app<ai-app-assistant>`, the *text* app and :ref:`other apps making use of the core Translation API<mt-consumer-apps>`. The *translate* app specifically runs only open source models and does so entirely on-premises. Nextcloud customers receive customer support upon request.
+
+The app currently supports the following languages:
+
+* English (en)
+* German (de)
+* French (fr)
+* Spanish (es)
+* Chinese (zh)
+
+As the models are entirely open source, the quality of translations may not be comparable to commercially available machine translation services.
+
+Requirements
+------------
+
+* x86 CPU
+* GNU lib C (musl is not supported)
+* This app does not support using GPU for processing and may thus not be performing ideally for long texts
+
+(*Note*: Nextcloud AIO is currently not supported due to it using musl)
+
+Space usage
+~~~~~~~~~~~
+
+ * ~1GB per language pair
+ * ~10GB in total
+
+Installation
+------------
+
+You can install the assistant app via the "Apps" page in Nextcloud, or by executing
+
+``
```

Comment:
Usually commands in Sphinx have such format:

```
You can install the assistant app via the "Apps" page in Nextcloud, or by executing :command:`occ`::

    occ app:enable translate
    

```    
This format automatically has "copy" button.

Ignore this comment if it not what you want )